src/auth/auth-errors.ts

```diff
@@ -1 +1,2 @@
-export const sessionTimedOutErrorMessage = 'Your session has expired.';
```

Comment:
UX question... maybe leave this as the notification title and put the lengthier explanation in the `detail` of `sessionExpirationProps`?

Reply:
Putting it in `detail` hides the info behind a button, brings up a modal, and prompts the user to refresh the page, which does absolutely nothing to remediate the notification. I'm working with UX to make sure the message is appropriate, both in length and in content.

Reply:
You're right. I was misremembering the current UI.

`message` (instead of `detail`) is the option for what I was thinking:

![Screenshot 2024-05-17 at 10 17 52 AM](https://github.com/DataBiosphere/terra-ui/assets/1156625/38d9de3d-835d-48f4-8445-99428b9f4dcf)